src/WebJobs.Script.WebHost/ProxyFunctionExecutor.cs

```diff
@@ -22,19 +22,87 @@ public class ProxyFunctionExecutor : IFuncExecutor
     {
         private readonly WebScriptHostManager _scriptHostManager;
         private readonly ISecretManager _secretManager;
+        private readonly string _httpPrefix;
+
         private WebHookReceiverManager _webHookReceiverManager;
 
         internal ProxyFunctionExecutor(WebScriptHostManager scriptHostManager, WebHookReceiverManager webHookReceiverManager, ISecretManager secretManager)
         {
             _scriptHostManager = scriptHostManager;
             _webHookReceiverManager = webHookReceiverManager;
             _secretManager = secretManager;
+
+            _httpPrefix = "api/";
+
+            if (_scriptHostManager.Instance != null)
+            {
+                var json = _scriptHostManager.Instance.ScriptConfig.HostConfig.HostConfigMetadata;
+
+                if (json != null && json["http"] != null && json["http"]["routePrefix"] != null)
```

Comment:
What about VS authored precompiled functions? In such cases, will the route be present in function.json?